hodor

```diff
@@ -44,21 +50,39 @@ class Hodor
     conf = YAML.load(ERB.new(File.read('.hodorfile')).result)
     @containers = conf['containers']
     @tasks = conf['tasks']
+    @hostmanager = conf['host-manager'] || "boot2docker"
+    if @hostmanager == "boot2docker"
+      @host = "boot2docker-vm"
+    elsif @hostmanager == "docker-machine"
+      @host = conf['host'] || ENV['DOCKER_MACHINE_NAME'] || 'default'
+    end
```

Comment:
Lets move parts of this logic back to initialize method. So if we know that by default we use boot2docker:

````
@hostmanager = "boot2docker"
@host = "boot2docker-vm"
@ssh_file = "~/.ssh/id_boot2docker"
````
Then inside `process_project_config` we just check if it's not default:

````
case conf['host-manager']
   when "docker-machine"
     @hostmanager = "docker-machine"
     @host = conf['host'] || ENV['DOCKER_MACHINE_NAME'] || 'default';
end
````